src/Apps/CollectorProfile/Routes/Saves2/Components/FourUpImageLayout.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Flex, Image, Spacer } from "@artsy/palette"
+import { prepareImageURLs } from "Apps/CollectorProfile/Routes/Saves2/Utils/prepareImageURLs"
+import { FC } from "react"
+import { cropped } from "Utils/resized"
+
+interface FourUpImageLayoutProps {
+  imageURLs: (string | null)[]
+}
+
+interface RowImageProps {
+  url: string | null
+}
+
+const LARGE_IMAGE_SIZE = 100
+const SMALL_IMAGE_SIZE = 58
+const IMAGE_OFFSET = "2px"
+
+export const FourUpImageLayout: FC<FourUpImageLayoutProps> = ({
+  imageURLs,
+}) => {
+  const preparedImageURLs = prepareImageURLs(imageURLs)
+
+  return (
+    <Box>
+      <Flex flexDirection="row">
+        <RowImage url={preparedImageURLs[0]} />
+        <Spacer x={IMAGE_OFFSET} />
+        <RowImage url={preparedImageURLs[1]} />
+      </Flex>
+
+      <Spacer y={IMAGE_OFFSET} />
+
+      <Flex flexDirection="row">
+        <RowImage url={preparedImageURLs[2]} />
+        <Spacer x={IMAGE_OFFSET} />
+        <RowImage url={preparedImageURLs[3]} />
+      </Flex>
+    </Box>
+  )
+}
+
+const RowImage: FC<RowImageProps> = ({ url }) => {
+  const SIZE = [SMALL_IMAGE_SIZE, LARGE_IMAGE_SIZE]
+
+  if (url === null) {
+    return (
+      <Box
+        width={SIZE}
+        height={SIZE}
+        bg="black5"
+        border="1px solid"
+        borderColor="black10"
+        aria-label="Image placeholder"
+      />
+    )
+  }
+
+  const image = cropped(url, {
+    width: LARGE_IMAGE_SIZE,
+    height: LARGE_IMAGE_SIZE,
+  })
+
+  return (
+    <Image
+      width={SIZE}
+      height={SIZE}
+      src={image.src}
+      srcSet={image.srcSet}
+      alt=""
```

Comment:
**question(non-blocking):** Should we pass something here? Like the title of the (artwork) image? Asking cause it might have impact on the SEO 🤔 (was reading [this](https://moz.com/learn/seo/alt-text))

Reply:
I don't think we need to worry about this, since this page is only available to logged in users (search bots will not have access to this page)

Reply:
Yeah, no impact on SEO. And the thing to remember about `alt` tags is that you don't want to duplicate any information that's already labelling the node. And so for a lot of our content, images are essentially decorative, since we don't have the ability to describe the actual images. Good `alt` text is **hard**.